### PR TITLE
Add loading and deleting on homepage

### PR DESCRIPTION
**Pull request by OlexG**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -54,8 +56,8 @@ export default function Index () {
   return (
     <main className="min-h-screen bg-white">
       <Header />
-      <div className="px-10">
-        <div className="flex flex-row gap-6 items-center">
+      <div className="px-10 w-screen">
```
Comment (reakunen) [inline pages/index.tsx:59]: couldnt u just do w-screen on the main ?

File: pages/index.tsx [hunk 2]
```diff
@@ -98,10 +100,14 @@ export default function Index () {
               phoneNumber={contact.phoneNumber ?? ''}
               index={i}
               email={contact.email ?? ''}
+              setIsLoading={setIsLoading}
             />
           </div>
           ))
-          : <div className="text-center text-lg font-light">Please login or add more contacts</div>}
+          : (!isLoading && <div className="text-center text-lg font-light">Please login or add more contacts</div>)}
```
Comment (reakunen) [inline pages/index.tsx:107]: could use isLoading ? (statement) : (statement) 
instead of using && twice

**Review by reakunen — APPROVED** [review]
very nice

**Comment by OlexG** [comment]
> Runs well! Frontend delete properly deletes from backend. well done olex!

Thanks!

**Comment by jeremiahliao** [comment]
everything looks good!

**Comment by evayktv** [comment]
LGTM

**Pull request merged**